uiowa/uiowa
# Add person schema to profile pages.

Resolves #5093.

While working on this, I realized that profiles functionality does not work in IE11 at all. Therefore, I used JS features like fetch and backticks which are not supported in IE11. Somewhat related: #4036



# How to test

- Pull down education.uiowa.edu.
- Go to the profiles directory config page.
- [Change the API key to this one.](https://iowaweb.slack.com/archives/C0164FNN1PV/p1647377146635319?thread_ts=1647377129.307609&cid=C0164FNN1PV)
- Navigate to https://education.uiowa.ddev.site/directory.
- Check that there is a `meta` element in the `head` element that has `name="description"` that describes the directory.
- Click on an individual person.
- Check that there is a `meta` element in the `head` element that has `name="description"` that describes that person.
- Check that there is a canonical meta element set.
- Check that there is a script tag in the head that has type `application/ld+json` and contains the appropriate person schema markup.
- Go back to the directory using the breadcrumbs. 
- Make sure the person-specific metadata is not set.
- Go back to a person. Do a full page reload and ensure metadata again.
